### DISCUSSION TITLE
Can't get teacher account reverified

### DISCUSSION THREAD

**Discussion by kroznai** [community/community#130261] [GitHub Education]
### Hi everyone,

I had a teacher account for the past few years, but it's expiring.  I tried to reverify, but have gotten rejected twice.  The application won't allow me to upload a document or image, only take a picture.  My district's teacher ID cards don't have years on them.  I've been trying to pull up my paystub on my phone and use my webcam to take a picture of my phone (which is a really irritating workaround, but again, the application will ONLY allow me to use my webcam to add an image).  For proof of why I'm not on campus (it is summertime), I've been pulling up the district calendar on my phone and using my webcam to take a photo of that.  These webcam photos of images on my phone are blurry and hard to read, which I'm guessing is why I keep getting rejected.  I'm not sure what else to do here.  I tried contacting a direct person, but github won't let me, as teacher applications are no longer manually reviewed. I need this teacher account to access my class's curriculum, which is hosted on github (but only accessible for teachers)!

**Comment by alexisseurin** [comment]
You can get help with GitHub Education here: https://support.github.com/contact/education

**Reply by Ygilany** [reply]
I have the same issue for over 2 weeks and they don't respond!
